backends/opt/ceed-opt-operator.c

```diff
@@ -42,29 +42,52 @@ static int CeedOperatorSetupFields_Opt(CeedQFunction qf, CeedOperator op, bool i
 
     if (eval_mode != CEED_EVAL_WEIGHT) {
       CeedCallBackend(CeedOperatorFieldGetElemRestriction(op_fields[i], &r));
-      Ceed ceed;
```

Comment:
This should be here because this `ceed` may be different than the `ceed` for the whole operator

Reply:
Ah I see, thanks! [f52c7b8](https://github.com/CEED/libCEED/pull/1168/commits/f52c7b8346769c8482ac5c1fee869e66a09109bc) should fix this and I think the renaming should clarify this for other developers hopefully.